pkg/arvo/lib/push-hook.hoon

```diff
@@ -1,15 +1,55 @@
+::  lib/push-hook/hoon: helper for creating a push hook
+::  
+::   lib/push-hook is a helper for automatically pushing data from a
+::   local store to the corresponding pull-hook on remote ships. It also
+::   proxies remote pokes to the store.
```

Comment:
> automatically pushing data from a local store to ... remote ships

This is why I don't like the "pull-push" phrasing. You're not so much _pushing_ as you are answering requests. Am I alone in thinking this is weird?

Reply:
I do think the listen/proxy dichotomy is clearer, but pull/push is clearer.

Reply:
I like having symmetrical, short names. Lmk if you have recommendations

Reply:
Symmetrical in what sense? Character length, or semantics?

I think that's less important than being clear about what these things do. Best alternative I got is hear/show.